Jenkinsfile.kola.aws

```diff
@@ -45,7 +45,7 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             utils.shwrap("""
             export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
             coreos-assembler init https://github.com/coreos/fedora-coreos-config
-            coreos-assembler buildprep --ostree s3://${params.S3_STREAM_DIR}/builds
+            coreos-assembler buildprep --ostree --build=${params.VERSION} s3://${params.S3_STREAM_DIR}/builds
```

Comment:
We also need to pass `--build` to the `cosa kola` commands below.

Reply:
I wasn't sure about this because the first one passes in `--aws-ami ${ami}` which I would think is all the information that is being gleaned from the builds directory. Is it ?

Reply:
The thing is that `cosa kola` today automatically passes `--cosa-build` to `kola`. So if we *don't* override it, I'm pretty sure it would fail entirely.

You're correct though that `kola run -p aws` today doesn't actually use the data from `meta.json`. But it will soon, and we'll be able to e.g. drop the whole `--aws-ami` switch and logic for it once we implement it: https://github.com/coreos/mantle/blob/d17c87d029ea905ae88b18d7e9fed7fd7a5e7968/cmd/kola/options.go#L204

Reply:
ok I think I've addressed this now